### ISSUE TITLE
"Validate Formula" link always fails in form editor

### ISSUE THREAD

**Issue by georgemandis** [snaptortoise/gravity-forms-minmax#1]
I believe the fix has something to do with implementing this filter:

[https://docs.gravityforms.com/gform_validation_error_form_editor/](https://docs.gravityforms.com/gform_validation_error_form_editor/)

In my testing so far it never seems to work. Not sure what I'm doing wrong.

**Comment by georgemandis** [comment]
Update on this issue:

There is a new JS filter in Gravity Forms 2.4.6.8 that will allow for overriding the result of the formula validation:

```
gform.addFilter('gform_is_valid_formula_form_editor', function (result, formula) {
// perform custom validation and set result to true or false
return result;
});
```

**Comment by georgemandis** [comment]
Fixed in v0.4.0
